litellm/litellm_core_utils/get_llm_provider_logic.py

```diff
@@ -86,6 +86,23 @@ def get_llm_provider(  # noqa: PLR0915
             if _is_non_openai_azure_model(model):
                 custom_llm_provider = "openai"
                 return model, custom_llm_provider, dynamic_api_key, api_base
+        elif model.split("/", 1)[0] == "nvidia" or model in litellm.nvidia_models:
+            api_base = (
+                api_base 
+                or get_secret("NVIDIA_API_BASE") 
```

Comment:
we're using `NVIDIA_BASE_URL` in other frameworks. `*_API_BASE` appears idiomatic for litellm. can we do both?

Reply:
also keep backward compat w/ `NVIDIA_NIM_API_BASE`